lib/rust/ensogl/component/grid-view/src/visible_area.rs

```diff
@@ -0,0 +1,123 @@
+//! A module with [`VisibleArea`] structure.
```

Comment:
a short info of what it is would be appreciated :)

Reply:
Actually, it's an outdated doc.